python/pylibcudf/pylibcudf/column.pyx

```diff
@@ -455,6 +455,8 @@ def is_c_contiguous(
         The boolean answer.
     """
 
+    if strides is None:
```

Comment:
I would be more inclined to put this case in `from_cuda_array_interface_obj` since a `None` strides is more of a definition of the interface as opposed to this function generally checking if an object is C-contiguous

Reply:
OK sure thing. Could you update the type annotations and docstring to note the meaning of `strides=None`?

Reply:
Yes, added in https://github.com/rapidsai/cudf/pull/18295/commits/b567b3b8ae7091b9772de5b7301c5b356baf38b9